SECURITY.md

```diff
@@ -0,0 +1,22 @@
+# Security Policy
+
+## Security Announcements
+
+Join the [kubernetes-security-announce] group for security and vulnerability announcements.
+
+You can also subscribe to an RSS feed of the above using [this link][kubernetes-security-announce-rss].
+
+## Reporting a Vulnerability
+
+Instructions for reporting a vulnerability can be found on the
+[Kubernetes Security and Disclosure Information] page.
+
+## Supported Versions
+
+Information about supported Kubernetes versions can be found on the
+[Kubernetes version and version skew support policy] page on the Kubernetes website.
```

Comment:
The Kubernetes version and version skew support policy is less relevant for this repo. More relevant would be [Releases and versioning](https://kops.sigs.k8s.io/welcome/releases/), as referenced in our README.

Reply:
@johngmyers  Sorry about that. I'm just pushing the same generic file to all our repos, but it makes sense for kops to reference its own policy as appropriate. How would you like to proceed? Do you think we should merge this and then kops maintainers can edit as you see fit in follow-on PRs? Or do you want to propose an alternate PR now and I'll close this? Or do you want to give me edits and I'll make them in this PR?